yjit/Cargo.toml

```diff
@@ -17,11 +17,15 @@ crate-type = ["staticlib"]
 # written rationale. Optional For development and testing purposes
 capstone = { version = "0.12.0", optional = true }
 
-[features]
 # NOTE: Development builds select a set of these via configure.ac
 # For debugging, `make V=1` shows exact cargo invocation.
+[features]
+# Support --yjit-dump-disasm and RubyVM::YJIT.disasm using libcapstone.
 disasm = ["capstone"]
-stats = []
+# Modify generated code for assertions. This is managed separately
+# from cfg!(debug_assertions) so that we can see disasm of the code
+# that would run in the release mode.
+runtime_assertions = []
```

Comment:
Just to clarify, `runtime_assertions` here would be things like the stack canary check? Anything else? Can you mention those specifically in the comment.

Also, could we rename `runtime_assertions` to `runtime_checks`, just to make it clear that this has nothing to do with `assert!` in C/Rust code?

Reply:
Right now it's only in two places: PC poison value for C method calls and stack canary. I mentioned them in the comment and renamed it to `runtime_checks` https://github.com/ruby/ruby/pull/11198/commits/b9529a187cdbf0e5b19b38a82849e1baf4b9a28a.